public/apps/configuration/panels/role-view/role-view.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import React, { Fragment, useEffect, useState } from 'react';
+
+import {
+  EuiBreadcrumbs,
+  EuiButton,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiSpacer,
+  EuiTab,
+  EuiTabs,
+  EuiTitle,
+} from '@elastic/eui';
+import { AppDependencies } from '../../../types';
+
+const tabs = [
+  {
+    id: 'permissions',
+    name: 'Permissions',
+    disabled: false,
+  },
+  {
+    id: 'users',
+    name: 'Users',
+    disabled: false,
+  },
+];
+
+interface RoleViewProps extends AppDependencies {
+  roleName: string;
+}
+
+export function RoleView(props: RoleViewProps) {
+  const [selectedTabId, setSelectedTabId] = useState('permissions');
+  const [breadcrumbs, setBreadcrumbs] = useState([]);
```

Comment:
I don't think the whole breadcrumbs need to be state. How about define a function that takes a role name and returns the breadcrumbs?

Reply:
Sure. Made a function